estate_account/__manifest__.py

```diff
@@ -0,0 +1,12 @@
+{
+    'name': "Real Estate Accounting Link",
+    'depends': [
+        'estate',
+        'account',
+    ],
+    'data': [
+    ],
+    'installable': True,
+    'application': False,
```

Comment:
```suggestion
    'application': False,
    'auto_install': True,
```
brigde modules typically contain functionally that we want to automatically "appear" when the dependencies are all installed